cdi/pom.xml

```diff
@@ -16,16 +16,16 @@
     SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.glassfish</groupId>
+        <groupId>org.wildfly.faces</groupId>
         <artifactId>mojarra-parent</artifactId>
-        <version>4.0.1-SNAPSHOT</version>
+        <version>4.0.0.SP01-SNAPSHOT</version>
```

Comment:
Just to check, this was based off 4.0.0-M6, right? I'm guessing there was just an extra commit in their branch that bumped to 4.0.1?